client/js/lounge.js

```diff
@@ -129,6 +129,11 @@ $(function() {
 				feedback.hide();
 			});
 		}
+
+		if (data.token) {
```

Comment:
I think this will make the user remember the login when changing their password, so need to add a check for that. I guess need to check if `token` exists in localstorage or something.